src/dispatch/case/views.py

```diff
@@ -188,6 +189,35 @@ def create_case(
     return case
 
 
+@router.post(
+    "/{case_id}/resources/conversation",
+    response_model=CaseRead,
+    summary="Creates conversation channel for an existing case.",
+)
+def create_case_channel(
+    db_session: DbSession,
+    case_id: PrimaryKey,
+    current_case: CurrentCase,
+    background_tasks: BackgroundTasks,
+):
+    """Creates conversation channel for an existing case."""
+
+    case = get(case_id=case_id, db_session=db_session)
+    case.dedicated_channel = True
+
+    participant_emails = [participant.individual.email for participant in current_case.participants]
```

Comment:
unrelated thought : i think we do this in quite a few places, potentially we could make this a property in the case/incident/participants model

Reply:
Done